cli/cmd/install.go

```diff
@@ -49,9 +49,15 @@ func installPlugin(cmd *cobra.Command, args []string) error {
 	}
 	sources := specReader.Sources
 	destinations := specReader.Destinations
-	opts := []managedplugin.Option{managedplugin.WithNoExec()}
+	opts := []managedplugin.Option{
+		managedplugin.WithNoExec(),
+		managedplugin.WithLogger(log.Logger),
+	}
 	if cqDir != "" {
-		opts = append(opts, managedplugin.WithDirectory(cqDir))
+		managedplugin.WithDirectory(cqDir)
```

Comment:
This is a no-op?

Reply:
Isn't it this?
```go
func WithDirectory(directory string) Option {
	return func(c *Client) {
		c.directory = directory
	}
}
```
How does this work if we're not using `Option`?

Reply:
There's a default https://github.com/cloudquery/plugin-pb-go/blob/f92d39c67536f41473abef7d6d9ebc30015c393e/managedplugin/plugin.go#L133

Reply:
That's for the Slack discussion I misunderstood you. I updated the PR